server/plugins/schema-plugin-api/src/main/java/com/oceanbase/odc/plugin/schema/api/ExternalTableExtensionPoint.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.plugin.schema.api;
+
+import java.sql.Connection;
+import java.util.List;
+
+import org.pf4j.ExtensionPoint;
+
+import com.oceanbase.tools.dbbrowser.model.DBObjectIdentity;
+
+/**
+ * @description:
+ * @author: zijia.cj
+ * @date: 2024/8/23 14:51
+ * @since: 4.3.3
+ */
+public interface ExternalTableExtensionPoint extends ExtensionPoint {
```

Comment:
there is no need to add a new extension point, we may add a new parameter on  `TableEntensionPoint#list(..., @NonNull List<DBObjectType> types)`